HabitRPG/Views/ToastView.swift

```diff
@@ -121,6 +121,14 @@ class ToastView: UIView {
         loadOptions()
     }
     
+    public convenience init(goldDiff: Float, background: ToastColor, duration: Double? = nil, delay: Double? = nil) {
```

Comment:
I experimented with editing the init that took in `goldDiff` and other changes but that seemed to make the init a bit unreadable...happy to make that init do it all though!

Reply:
No, this looks fine.